README.md

```diff
@@ -1,7 +1,39 @@
-Should be put alongside the engine repo:
+# test-gfx
+
+a light-weight repo used to test the feature of the _gfx_ module of cocos-creator.
+
+## Build
+
+_test_gfx_ should be put alongside the engine repo:
 
 <pre>
 ├── cocos-engine (<a href="https://github.com/cocos/cocos-engine/tree/v3.5.0">3.5.0</a>)
 │   └── native/external (<a href="https://github.com/cocos/cocos-engine-external/tree/v3.5">3.5</a>)
 └── test-gfx (<a href="https://github.com/cocos-creator/test-gfx/tree/master">master</a>)
 </pre>
+
+### web
+
+Enter ./engine folder and run the following commands
+```bash
+    $ ./util.sh update       # build the project.
```

Comment:
command is update, the operation is building?

Reply:
Yes, update will build the project and then copy the result to `engine/web-desktop`.